test-data/unit/check-intEnum-arguments.test

```diff
@@ -0,0 +1,7 @@
+[case testIntEnumArguments]
```

Comment:
Instead of adding a new test file, can you add the test case to `test-data/unit/check-enum.test` instead? Currently the test case won't be run, I think.

Reply:
Ok, will do!